src/main/scala/devices/tilelink/CLINT.scala

```diff
@@ -109,6 +109,14 @@ trait CanHavePeripheryCLINT { this: BaseSubsystem =>
     val clint = LazyModule(new CLINT(params, cbus.beatBytes))
     LogicalModuleTree.add(logicalTreeNode, clint.logicalTreeNode)
     clint.node := tlbus.coupleTo("clint") { TLFragmenter(tlbus) := _ }
+
+    // Override the implicit clock and reset -- could instead include a clockNode in the clint, and make it a RawModuleImp?
+    InModuleBody {
+      clint.module.clock := tlbus.module.clock
+      clint.module.reset := tlbus.module.reset
+    }
```

Comment:
You have done some of that already in another PR no? If you want i can fold that in.

Reply:
This is making me reconsider how I was doing that slightly, but I think it would be best done as a follow on on top of this, and so this is fine as is